js/Game.js

```diff
@@ -21,7 +21,12 @@ class Game {
   }
 
   changeTurn() {
-    this.turn++
+    debugger
+    this.playerIndex++;
```

Comment:
If playerIndex is 2 (player 3), then playerIndex will change back to 0 (player 1)

Reply:
Nice. Seems simple enough.